examples/cua/legacy_report_fetcher/portal/server.py

```
@@ -0,0 +1,296 @@
+        username = fields.get("username", [""])[0]
+        password = fields.get("password", [""])[0]
+        if username == USERNAME and password == PASSWORD:
+            self.send_response(HTTPStatus.SEE_OTHER)
+            self.send_header("Location", "/reports")
+            self.send_header("Set-Cookie", f"{SESSION_COOKIE}; Path=/; SameSite=Lax")
+            self.end_headers()
+            return
+
+        self._send_login(error="Invalid username or password.")
+
+    def log_message(self, format: str, *args: object) -> None:
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)